automation_tools/satellite6/upgrade/satellite.py

```diff
@@ -66,7 +66,7 @@ def satellite6_setup(os_version):
     # Set satellite hostname in fabric environment
     env['satellite_host'] = sat_host
     logger.info('Satellite {} is ready for Upgrade!'.format(sat_host))
-    if os.environ.get('RUN_EXISTANCE_TESTS'):
+    if os.environ.get('RUN_EXISTANCE_TESTS').lower() == 'true':
```

Comment:
What if we migrate jenkins and for some reason that param is forgot to be added? That is the point I had in mind when suggesting this. Also if someone wants to test locally he/she may forget to set this up.